GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -32,6 +32,17 @@ NS_ASSUME_NONNULL_BEGIN
 @class GIDVerifiableAccountDetail;
 @class GIDVerifiedAccountDetailResult;
 
+/// The error domain for `NSError`s returned by the Google Sign-In SDK.
+extern NSErrorDomain const kGIDVerifyErrorDomain;
+
+/// A list of potential error codes returned from the Google Sign-In SDK.
+typedef NS_ERROR_ENUM(kGIDVerifyErrorDomain, GIDVerifyErrorCode) {
+  /// Indicates an unknown error has occurred.
+  kGIDVerifyErrorCodeUnknown = 1,
```

Comment:
I think I prefer not having the `k` prefix in the case names. Those aren't in the [style guide](https://google.github.io/styleguide/objcguide.html#constants).

Reply:
Done.